code/datums/components/uplink.dm

```diff
@@ -359,6 +359,22 @@
 	if(ismob(master.loc))
 		interact(null, master.loc)
 
+/datum/component/uplink/proc/new_message(datum/source, user, message, channel)
+	SIGNAL_HANDLER
+
+	if(channel != RADIO_CHANNEL_UPLINK)
+		return
+
+	. = COMPONENT_CANNOT_USE_RADIO // we're using uplink channel, cut all communication
```

Comment:
return COMPONENT_CANNOT_USE_RADIO in relevant places instead.

https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-